src/Output/Output_L1Error.cpp

```diff
@@ -319,20 +331,25 @@ void WriteFile( void (*AnalFunc_Flu)( real fluid[], const double x, const double
    MHD_GetCellCenteredBFieldInPatch( Nume+NCOMP_TOTAL, lv, PID, i, j, k, amr->MagSg[lv] );
 #  endif
 
-
-// convert total energy to pressure
+// get pressure and temperature
 #  if ( MODEL == HYDRO )
    const bool  CheckMinPres_No = false;
+   const bool  CheckMinTemp_No = false;
 #  ifdef MHD
    const real *B_Nume          = Nume + NCOMP_TOTAL;
    const real  Emag_Nume       = (real)0.5*( SQR(B_Nume[MAGX]) + SQR(B_Nume[MAGY]) + SQR(B_Nume[MAGZ]) );
 #  else
    const real  Emag_Nume       = NULL_REAL;
 #  endif
 
-   Nume[ENGY] = Hydro_Con2Pres( Nume[DENS], Nume[MOMX], Nume[MOMY], Nume[MOMZ], Nume[ENGY], Nume+NCOMP_FLUID,
-                                CheckMinPres_No, NULL_REAL, Emag_Nume,
-                                EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table, NULL );
+   const real  Pres_Nume       = Hydro_Con2Pres( Nume[DENS], Nume[MOMX], Nume[MOMY], Nume[MOMZ], Nume[ENGY], Nume+NCOMP_FLUID,
+                                                 CheckMinPres_No, NULL_REAL, Emag_Nume,
+                                                 EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table, NULL );
+   const real  Temp_Nume       = Hydro_Con2Temp( Nume[DENS], Nume[MOMX], Nume[MOMY], Nume[MOMZ], Nume[ENGY], Nume+NCOMP_FLUID,
+                                                 CheckMinTemp_No, NULL_REAL, Emag_Nume,
+                                                 EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table );
```

Comment:
**_EoS_DensEint2Pres_CPUPtr_** should be changed to **_EoS_DensEint2Temp_CPUPtr_**

Reply:
Good catch! Fixed.